homeassistant/components/samsungtv/device_trigger.py

```diff
@@ -0,0 +1,106 @@
+"""Provides device automations for Samsung TV."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant.components.device_automation import DEVICE_TRIGGER_BASE_SCHEMA
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    CONF_DEVICE_ID,
+    CONF_DOMAIN,
+    CONF_ENTITY_ID,
+    CONF_PLATFORM,
+    CONF_TYPE,
+)
+from homeassistant.core import CALLBACK_TYPE, Event, HassJob, HomeAssistant, callback
+from homeassistant.helpers import config_validation as cv, entity_registry
+from homeassistant.helpers.trigger import TriggerActionType, TriggerInfo
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN, EVENT_TURN_OFF, EVENT_TURN_ON
+
+TRIGGER_TYPES = {"turn_on", "turn_off"}
+
+TRIGGER_SCHEMA = DEVICE_TRIGGER_BASE_SCHEMA.extend(
+    {
+        vol.Required(CONF_ENTITY_ID): cv.entity_id,
+        vol.Required(CONF_TYPE): vol.In(TRIGGER_TYPES),
+    }
+)
+
+
+async def async_get_triggers(
+    hass: HomeAssistant, device_id: str
+) -> list[dict[str, str]]:
+    """List device triggers for Samsung TVs."""
+    registry = entity_registry.async_get(hass)
+    triggers = []
```

Comment:
```suggestion
    triggers: list[dict[str, str]] = []
```

Reply:
Done. I thing it's logical to have `turn_on` if we have `turn_off` at least for consistency purposes.